tinydraw.go

```diff
@@ -4,6 +4,7 @@ import (
 	"image/color"
 
 	"tinygo.org/x/drivers"
+	"github.com/pkg/errors"
```

Comment:
pulling in pkg/errors depends on runtime.Callers:

```
root@261de9f4d161:/in# tinygo build -target gameboy-advance -o withbugfix.gba .
# github.com/pkg/errors
../root/go/src/github.com/pkg/errors/stack.go:166:15: Callers not declared by package runtime
../root/go/src/github.com/pkg/errors/stack.go:39:16: fn.FileLine undefined (type *runtime.Func has no field or method FileLine)
../root/go/src/github.com/pkg/errors/stack.go:28:16: fn.FileLine undefined (type *runtime.Func has no field or method FileLine)
```

It might be better to just panic or do nothing if the rectangle isn't large enough.